helm_resource/Tiltfile

```diff
@@ -73,8 +73,8 @@ def helm_resource(
       # NOTE(nick): This is a PITA because it's hard to separate
       # out the repository from the tag.
       flags.extend([
-        '--set', '%s=$(echo "$TILT_IMAGE_%s" | sed -r \'s,:[^/]*$,,\')' % (shlex.quote(key[0]), i),
-        '--set', '%s=$(echo "$TILT_IMAGE_%s" | sed -r \'s,^.*:([^/]*)$,\\1,\')' % (shlex.quote(key[1]), i),
+        '--set', '%s=$(echo "$TILT_IMAGE_%s" | sed -e \'s,:[^/]*$,,\')' % (shlex.quote(key[0]), i),
+        '--set', '%s=$(echo "$TILT_IMAGE_%s" | sed -e \'s,^.*:\\([^/]*\\)$,\\1,\')' % (shlex.quote(key[1]), i),
```

Comment:
do you need the `-e` at all? i think this _might_ work with the "default" regexp mode (though i could be wrong)

Reply:
Yeah, it's unnecessary - dropped in 1c01180